### DISCUSSION TITLE
Does emf.cloud support vscode?

### DISCUSSION THREAD

**Discussion by luxingxiao** [eclipse-emfcloud/emfcloud#175] [All other] [answered]
Is it possible to develop emf.cloud as a vscode extension.

**Comment by ndoschek** [comment, marked as answer]
Hi @luxingxiao,

Basically, EMF.cloud is an umbrella project for different [components](https://github.com/eclipse-emfcloud/emfcloud#components).
There is no dedicated VSCode integration yet for our components (where applicable).

Currently it is not very high up in the backlog, but if there is strong interest for VSCode integrations (where applicable) and potentially an opportunity to collaborate, e.g. sponsored development (https://www.eclipse.org/emfcloud/support) and other contributions, we'd be very happy to work on that together!
